tests/xtro-sharpie/EnumCheck.cs

```diff
@@ -57,8 +57,18 @@ public override void VisitEnumDecl (EnumDecl decl, VisitKind visitKind)
 			if (!enums.TryGetValue (mname, out var type)) {
 				Log.On (framework).Add ($"!missing-enum! {name} not bound");
 				return;
-			} else
-				enums.Remove (mname);
+			} else {
+            	var enumMembersInXamarin = type.Fields.Select (e => name.Replace ("Code","") + e.Name);
```

Comment:
`name.Replace ("Code","")` should not be needed

The earlier logic has already figured out the managed and native name (and `Code` is only one of many difference that can happen).